debezium-connector-jdbc/README.md

```diff
@@ -81,6 +81,47 @@ An example configuration:
      collection.naming.strategy=io.debezium.sink.naming.DefaultCollectionNamingStrategy
      column.naming.strategy=io.debezium.connector.jdbc.naming.DefaultColumnNamingStrategy
 
+### Enhanced Naming Strategies
```

Comment:
This should be reworked to discuss this from the perspective of the transformations, not the configuration fields we want to remove in favor of the transformations.